.github/workflows/firebase-hosting-merge.yml

```diff
@@ -19,12 +19,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: echo "REACT_APP_BACKEND_URL=${{ secrets.REACT_APP_BACKEND_URL_PROD }}" > .env
+      - run: echo "REACT_APP_BACKEND_URL=${{ secrets.DEV_BACKEND_URL }}" > .env
+      - run: echo "REACT_APP_OAUTH_CLIENT_ID=${{ secrets.DEV_OAUTH_CLIENT_ID }}" >> .env
```

Comment:
Having some issues with the OAuth client initialization in the frontend, will investigate and fix separately

![image](https://user-images.githubusercontent.com/30205929/148718087-a1107e14-d127-423e-a527-7ee72e54c70a.png)

Reply:
Got a more detailed error message and tried to fix by adding the preview deploy URL as an authorized origin in the GCP console but that didn't seem to work...I wonder if it's because this OAuth client is currently set in testing mode

```json
{
  "error":"idpiframe_initialization_failed",
  "details":"Not a valid origin for the client: https://uw-blueprint-starter-code--pr138-sherry-deployment-pr-or67osgd.web.app has not been registered for client ID 997846099734-rneoi95fjccn3s192so93vp4vr9u6s41.apps.googleusercontent.com. Please go to https://console.developers.google.com/ and register this origin for your project's client ID."
}
```